backend/qemu.pm

```diff
@@ -792,7 +792,10 @@ sub start_qemu {
         }
 
         if ($arch_supports_boot_order) {
-            if ($vars->{PXEBOOT}) {
+            if ($vars->{PXEBOOT} // '' eq 'once') {
```

Comment:
I hope this works without parentheses …

Reply:
D'oh...I didn't test it, didn't notice you had parentheses in yours. Let me go test in in a test script real quick.

...

well, seems fine according to this test script:

    #!/bin/perl

    use warnings;
    use strict;

    my $vars = {foo => 'foo'};
    print "True!\n" if ($vars->{bar} // '' eq 'once');
    print "True 2!\n" if ($vars->{foo} // '' eq 'foo');

it runs without complaints and prints 'True 2!'.